elastic/kibana
# [ObsUx] [Controls] Add popover message in the control title

Closes #172046

## Summary
This PR adds a control_title component inside infra controls usage to give us the flexibility to use help messages in the controls.

![image](https://github.com/elastic/kibana/assets/14139027/fe949168-e5af-42a3-9721-18423b606fdb)

### Solution
Inside the control_title component, there is a definition of `helpMessages` used to add a help message in the infra control title matched by the control id. The message could be a string or string & link:
 - "Message"
   
<img width="428" alt="image" src="https://github.com/elastic/kibana/assets/14139027/5199bfb5-c676-42ae-85cd-78d1e26a6adc">

 - "Message" + Link 
  
<img width="408" alt="image" src="https://github.com/elastic/kibana/assets/14139027/d15a838c-ba59-4f10-82c6-0bedab33feb3">

## Testing 
- Go to Infrastructure -> Hosts
   - Syntrace can be used to get some services, for example, use: `node scripts/synthtrace --clean simple_trace.ts`
- Check the service name control and interact with it
   - Check help message 
   - Filter by service

   https://github.com/elastic/kibana/assets/14139027/e6aed1ca-29f1-47c8-9589-bea26a791598
